CONTRIBUTING.md

```diff
@@ -16,6 +16,14 @@ This document is an active work in progress! But, we hope you'll find some usefu
 
 ## Types of Contributions
 
+There are lots of ways to contribute to the Astro Docs website! 
+
+The Astro Docs website is ... an Astro website! Maintaining it requires not only written content but also maintaining Astro code and addressing a11y, CSS, UI, and UX concerns. We also make our documentation available in several languages, so we need help translating the entire site.
+
+You can also make a huge contribution by getting involved by leaving review comments on PRs, adding ideas in existing GitHub Issues and Discussions and participating in our "Pinned" issue maintenance tasks! 
```

Comment:
```suggestion
You can also make a huge contribution by getting involved by leaving review comments on [PRs](https://github.com/withastro/docs/pulls), adding ideas in existing GitHub [Issues](https://github.com/withastro/docs/issues) and [Discussions](https://github.com/withastro/docs/discussions) and participating in our "Pinned" issue maintenance tasks! 
```

maybe it's a good idea to link the real GitHub issues and discussion here?

Reply:
Thanks! I had these when I had more bullet points explaining each one below, but then didn't want it to get too wordy so I scrapped the extra content... and lost the links in the process!